src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesControl.xaml.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Input;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    /// <summary>
+    /// Interaction logic for ConfigurationFilesControl.xaml
+    /// </summary>
+    public partial class ConfigurationFilesControl : UserControl
+    {
+        private ConfigurationFilesViewModel _viewModel;
+
+        public ConfigurationFilesControl(ConfigurationFilesViewModel configurationFiles)
+        {
+            InitializeComponent();
+            _viewModel = configurationFiles;
+            DataContext = configurationFiles;
+        }
+
+        private void ListViewItem_MouseDoubleClick(object sender, MouseButtonEventArgs e)
+        {
+            _openButton.Command.Execute(null);
+        }
+
+        internal void InitializeOnActivated()
+        {
+            _viewModel.SetConfigPaths();
+        }
+
+        private void Hyperlink_RequestNavigate(object sender, System.Windows.Navigation.RequestNavigateEventArgs e)
+        {
+            if (sender is Hyperlink)
+            {
+                UIUtility.LaunchExternalLink(e.Uri);
```

Comment:
As per the comment I left previously

```suggestion
                UIUtility.LaunchExternalLink(e.Uri.AbsoluteUri);
```

Reply:
I tried this but `LaunchExternalLink` takes a Uri argument, and `AbsoluteUri` is a string. I think it's OK to leave as-is because `LaunchExternalLink` calls `Process.Start(url.AbsoluteUri)`, so it ends up using `AbsoluteUri` anyway.

Reply:
Good catch! In that case, passing the Uri is the best. Thanks!